### ISSUE TITLE
Bento init fails due to import shadowing

### ISSUE THREAD

**Issue by ievans** [returntocorp/bento#256]
## Overview
Running bento init on apache airflow fails

## Current Behavior
```
> bento init
...
bandit․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․|■■■■■■■■■□| 00:02, 🍜 Setting upTraceback (most recent call last):
  File "/usr/local/bin/virtualenv", line 10, in <module>□| 00:02, 🍜 Setting up
    sys.exit(main())․․․․․․․․․․․․․․․․․․․․․․․․․․|■■■■■■■■■■| 00:00, 🍱 Done
  File "/usr/local/lib/python3.7/site-packages/virtualenv.py", line 870, in main
    symlink=options.symlink,․․․․․․․․․․․․․․․․․․|■■■■■■■■■□| 00:02, 🍜 Setting up
  File "/usr/local/lib/python3.7/site-packages/virtualenv.py", line 1173, in create_environment
    install_wheel(to_install, py_executable, search_dirs, download=download)
  File "/usr/local/lib/python3.7/site-packages/virtualenv.py", line 1019, in install_wheel
    _install_wheel_with_search_dir(download, project_names, py_executable, search_dirs)
  File "/usr/local/lib/python3.7/site-packages/virtualenv.py", line 1110, in _install_wheel_with_search_dir
    call_subprocess(cmd, show_stdout=False, extra_env=env, stdin=script)
  File "/usr/local/lib/python3.7/site-packages/virtualenv.py", line 963, in call_subprocess
    raise OSError("Command {} failed with error code {}".format(cmd_desc, proc.returncode))
OSError: Command /Users/ine/.bento/venv/flake8/bin/python3.7 - setuptools pip wheel failed with error code 1
Traceback (most recent call last):
  File "/usr/local/bin/virtualenv", line 10, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.7/site-packages/virtualenv.py", line 870, in main
    symlink=options.symlink,
  File "/usr/local/lib/python3.7/site-packages/virtualenv.py", line 1173, in create_environment
    install_wheel(to_install, py_executable, search_dirs, download=download)
  File "/usr/local/lib/python3.7/site-packages/virtualenv.py", line 1019, in install_wheel
    _install_wheel_with_search_dir(download, project_names, py_executable, search_dirs)
  File "/usr/local/lib/python3.7/site-packages/virtualenv.py", line 1110, in _install_wheel_with_search_dir
    call_subprocess(cmd, show_stdout=False, extra_env=env, stdin=script)
  File "/usr/local/lib/python3.7/site-packages/virtualenv.py", line 963, in call_subprocess
    raise OSError("Command {} failed with error code {}".format(cmd_desc, proc.returncode))
OSError: Command /Users/ine/.bento/venv/click/bin/python3.7 - setuptools pip wheel failed with error code 1
bandit․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․|□□□□□□□□□□| 00:03, 🍜 Setting upTraceback (most recent call last):
  File "/usr/local/bin/virtualenv", line 10, in <module>□| 00:03, 🍜 Setting up
    sys.exit(main())
  File "/usr/local/lib/python3.7/site-packages/virtualenv.py", line 870, in main
    symlink=options.symlink,․․․․․․․․․․․․․․․․․․|□□□□□□□□□□| 00:03, 🍜 Setting up
  File "/usr/local/lib/python3.7/site-packages/virtualenv.py", line 1173, in create_environment
    install_wheel(to_install, py_executable, search_dirs, download=download)
  File "/usr/local/lib/python3.7/site-packages/virtualenv.py", line 1019, in install_wheel
    _install_wheel_with_search_dir(download, project_names, py_executable, search_dirs)
  File "/usr/local/lib/python3.7/site-packages/virtualenv.py", line 1110, in _install_wheel_with_search_dir
    call_subprocess(cmd, show_stdout=False, extra_env=env, stdin=script)
  File "/usr/local/lib/python3.7/site-packages/virtualenv.py", line 963, in call_subprocess
    raise OSError("Command {} failed with error code {}".format(cmd_desc, proc.returncode))
OSError: Command /Users/ine/.bento/venv/bandit/bin/python3.7 - setuptools pip wheel failed with error code 1
bandit․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․|■□□□□□□□□□| 00:03, 🍜 Setting upTraceback (most recent call last):
  File "/usr/local/bin/virtualenv", line 10, in <module>□| 00:03, 🍜 Setting up
    sys.exit(main())
  File "/usr/local/lib/python3.7/site-packages/virtualenv.py", line 870, in main
    symlink=options.symlink,․․․․․․․․․․․․․․․․․․|■□□□□□□□□□| 00:03, 🍜 Setting up
  File "/usr/local/lib/python3.7/site-packages/virtualenv.py", line 1173, in create_environment
    install_wheel(to_install, py_executable, search_dirs, download=download)
  File "/usr/local/lib/python3.7/site-packages/virtualenv.py", line 1019, in install_wheel
    _install_wheel_with_search_dir(download, project_names, py_executable, search_dirs)
  File "/usr/local/lib/python3.7/site-packages/virtualenv.py", line 1110, in _install_wheel_with_search_dir
    call_subprocess(cmd, show_stdout=False, extra_env=env, stdin=script)
  File "/usr/local/lib/python3.7/site-packages/virtualenv.py", line 963, in call_subprocess
    raise OSError("Command {} failed with error code {}".format(cmd_desc, proc.returncode))
OSError: Command /Users/ine/.bento/venv/flask/bin/python3.7 - setuptools pip wheel failed with error code 1
bandit․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․|■■□□□□□□□□| 00:03, 🍜 Setting upTraceback (most recent call last):
  File "/usr/local/bin/virtualenv", line 10, in <module>□| 00:03, 🍜 Setting up
    sys.exit(main())
  File "/usr/local/lib/python3.7/site-packages/virtualenv.py", line 870, in main
    symlink=options.symlink,․․․․․․․․․․․․․․․․․․|■■□□□□□□□□| 00:03, 🍜 Setting up
  File "/usr/local/lib/python3.7/site-packages/virtualenv.py", line 1173, in create_environment
    install_wheel(to_install, py_executable, search_dirs, download=download)
  File "/usr/local/lib/python3.7/site-packages/virtualenv.py", line 1019, in install_wheel
    _install_wheel_with_search_dir(download, project_names, py_executable, search_dirs)
  File "/usr/local/lib/python3.7/site-packages/virtualenv.py", line 1110, in _install_wheel_with_search_dir
    call_subprocess(cmd, show_stdout=False, extra_env=env, stdin=script)
  File "/usr/local/lib/python3.7/site-packages/virtualenv.py", line 963, in call_subprocess
    raise OSError("Command {} failed with error code {}".format(cmd_desc, proc.returncode))
OSError: Command /Users/ine/.bento/venv/requests/bin/python3.7 - setuptools pip wheel failed with error code 1
```

## Steps to Reproduce

1. Run bento init on fresh apache airflow clone
2. Init exits with 0 despite these failures, but then `bento check --all` fails:

```
✘ Error while running r2c.requests: Command '['python3', '-m', 'pip', 'list', '
--format', 'json']' returned non-zero exit status 1.
Traceback (most recent call last):
  File "/usr/local/Cellar/python/3.7.5/Frameworks/Python.framework/Versions/3.7/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/local/Cellar/python/3.7.5/Frameworks/Python.framework/Versions/3.7/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/Users/ine/.bento/venv/requests/lib/python3.7/site-packages/pip/__main__.py", line 16, in <module>
    from pip._internal import main as _main  # isort:skip # noqa
  File "/Users/ine/.bento/venv/requests/lib/python3.7/site-packages/pip/_internal/__init__.py", line 40, in <module>
    from pip._internal.cli.autocompletion import autocomplete
  File "/Users/ine/.bento/venv/requests/lib/python3.7/site-packages/pip/_internal/cli/autocompletion.py", line 8, in <module>
    from pip._internal.cli.main_parser import create_main_parser
  File "/Users/ine/.bento/venv/requests/lib/python3.7/site-packages/pip/_internal/cli/main_parser.py", line 11, in <module>
    from pip._internal.commands import (
  File "/Users/ine/.bento/venv/requests/lib/python3.7/site-packages/pip/_internal/commands/__init__.py", line 6, in <module>
    from pip._internal.commands.completion import CompletionCommand
  File "/Users/ine/.bento/venv/requests/lib/python3.7/site-packages/pip/_internal/commands/completion.py", line 6, in <module>
    from pip._internal.cli.base_command import Command
  File "/Users/ine/.bento/venv/requests/lib/python3.7/site-packages/pip/_internal/cli/base_command.py", line 29, in <module>
    from pip._internal.req.constructors import (
  File "/Users/ine/.bento/venv/requests/lib/python3.7/site-packages/pip/_internal/req/__init__.py", line 5, in <module>
    from .req_install import InstallRequirement
  File "/Users/ine/.bento/venv/requests/lib/python3.7/site-packages/pip/_internal/req/req_install.py", line 16, in <module>
    from pip._vendor.pep517.wrappers import Pep517HookCaller
  File "/Users/ine/.bento/venv/requests/lib/python3.7/site-packages/pip/_vendor/pep517/wrappers.py", line 5, in <module>
    from subprocess import check_call
ImportError: cannot import name 'check_call' from 'subprocess' (.../airflow/subprocess.py)
```

## Possible Solution
> Do you have an idea for how to address the issue or feedback?

## Additional Context
Bento version (run `bento --version`): bento/0.8.0
Operation system (e.g. Ubuntu 18.04): Mac OSX
Any other context that would help us reproduce or understand the issue:

**Comment by DrewDennison** [comment]
![image](https://user-images.githubusercontent.com/1011067/73221844-baf62180-4116-11ea-8b21-45b10493eec1.png)

What commit of airflow? `9a04013b0e40b0d744ff4ac9f008491806d60df2` works ok for me

**Comment by arashd** [comment]
I wasn't able to reproduce this on airflow@9a04013b, but likely due to `bento/extra/python_tool.py:60`

**Comment by ievans** [comment]
@DrewDennison @arashd I triaged a little further and realized I had added a `subprocess.py` file.

It appears that just doing `touch subprocess.py` in airflow root is enough to choke our processing.

```
✘ Error while running r2c.requests: Command '['python3', '-m', 'pip', 'list', '
--format', 'json']' returned non-zero exit status 1.
Traceback (most recent call last):
  File "/usr/local/Cellar/python/3.7.5/Frameworks/Python.framework/Versions/3.7/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/local/Cellar/python/3.7.5/Frameworks/Python.framework/Versions/3.7/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/Users/ine/.bento/venv/requests/lib/python3.7/site-packages/pip/__main__.py", line 19, in <module>
    sys.exit(_main())
  File "/Users/ine/.bento/venv/requests/lib/python3.7/site-packages/pip/_internal/cli/main.py", line 73, in main
    command = create_command(cmd_name, isolated=("--isolated" in cmd_args))
  File "/Users/ine/.bento/venv/requests/lib/python3.7/site-packages/pip/_internal/commands/__init__.py", line 96, in create_command
    module = importlib.import_module(module_path)
  File "/Users/ine/.bento/venv/requests/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/ine/.bento/venv/requests/lib/python3.7/site-packages/pip/_internal/commands/list.py", line 13, in <module>
    from pip._internal.cli.req_command import IndexGroupCommand
  File "/Users/ine/.bento/venv/requests/lib/python3.7/site-packages/pip/_internal/cli/req_command.py", line 21, in <module>
    from pip._internal.req.constructors import (
  File "/Users/ine/.bento/venv/requests/lib/python3.7/site-packages/pip/_internal/req/__init__.py", line 11, in <module>
    from .req_file import parse_requirements
  File "/Users/ine/.bento/venv/requests/lib/python3.7/site-packages/pip/_internal/req/req_file.py", line 25, in <module>
    from pip._internal.req.constructors import (
  File "/Users/ine/.bento/venv/requests/lib/python3.7/site-packages/pip/_internal/req/constructors.py", line 28, in <module>
    from pip._internal.req.req_install import InstallRequirement
  File "/Users/ine/.bento/venv/requests/lib/python3.7/site-packages/pip/_internal/req/req_install.py", line 17, in <module>
    from pip._vendor.pep517.wrappers import Pep517HookCaller
  File "/Users/ine/.bento/venv/requests/lib/python3.7/site-packages/pip/_vendor/pep517/wrappers.py", line 6, in <module>
    from subprocess import check_call, check_output, STDOUT
ImportError: cannot import name 'check_call' from 'subprocess' (./airflow/subprocess.py)
```

**Comment by DrewDennison** [comment]
I think this is why we always need to do imports from root. e.g. `from subprocess import check_call, check_output, STDOUT` should really be `from bento.subprocess import check_call, check_output, STDOUT`

**Comment by dlukeomalley** [comment]
@DrewDennison Is that a kick non-controversial fix to push in 0.8.1?

**Comment by ievans** [comment]
Reporter (me) was doing dumb things with Python path. My mistake!
